frontend/app/src/components/home/profile/Accounts.svelte

```diff
@@ -198,21 +235,40 @@
         th {
             @include font(book, normal, fs-70);
             padding-bottom: $sp4;
-            font-weight: 700;
 
             &.token-header {
+                font-weight: 700;
                 text-align: left;
             }
 
             &.balance-header {
-                padding-right: 28px;
-                text-align: right;
+                @include font-size(fs-60);
+                text-transform: uppercase;
             }
         }
 
         td {
             vertical-align: middle;
             padding-bottom: $sp3;
+
+            &.manage-col {
+                width: 0;
+            }
+
+            &.total {
+                @include font(bold, normal, fs-100, 22);
+                padding-right: 30px;
+                text-align: right;
+            }
+        }
+
+        tr.total {
+            color: var(--disabledTxt);
```

Comment:
I'd say `var(--txt-light)` is a better. They are probably the same, but seems safer for theme-ing.